app/components/ui/Navigation/DropdownSubmenu.tsx

```diff
@@ -0,0 +1,66 @@
+import React from "react";
+import { Link } from "react-router-dom";
+import navStyles from "./DesktopNavigation.module.scss";
+import categoryStyles from "./CategoryDropdown.module.scss";
+
+// This component is the dropdown submenu used by both DesktopMenuItems and CategoryDropdown, created in service of DRY code
+
+const DropdownSubmenu = ({
```

Comment:
I don't quite understand why this component is called a "submenu" vs a "menu." To me it reads like a _dropdown menu_ that isn't _necessarily_ a subordinate to another menu.

Reply:
I guess I was thinking because the navigation is the menu that this is a submenu, but if i can pull this all into one component, we can call it a dropdown menu